app/src/main/java/de/danoeh/antennapod/fragment/CategoriesFragment.java

```diff
@@ -0,0 +1,71 @@
+package de.danoeh.antennapod.fragment;
+
+import android.content.res.Resources;
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.GridView;
+import java.util.ArrayList;
+import java.util.Arrays;
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+
+public class CategoriesFragment extends Fragment {
+
+
+    public static final String TAG = "CategoriesFragment";
+    private ArrayAdapter listAdapter;
+    private GridView gridView;
```

Comment:
Isn't a [GridView](https://developer.android.com/guide/topics/ui/layout/gridview.html) supposed to show the elements in "squares"? Right now I see them as a list which is what we wanted to have. Why use a GridView here for the categories?

Reply:
Thank for pointing this out.